/**:
  ros__parameters:
    cam0:
      T_cam_imu: 
        [-0.999765, -0.021276, -0.004171, 0.0599907,
         0.021353, -0.999586, -0.019275, -7.93233e-05,
         -0.003759, -0.019359, 0.999806, -0.002,
         0.0,       0.0,       0.0,      1.0]
      camera_model: pinhole
      distortion_coeffs: [0.549546, 0.296002, -0.857166, 0.505325]
      distortion_model: equidistant
      intrinsics: [245.1435, 245.447, 321.5555, 213.2205]
      resolution: [640, 400]
      timeshift_cam_imu: 0.0
    cam1:
      T_cam_imu:
        [-0.999833, -0.0142211, -0.0115195, -0.0599733,
		     0.014361, -0.999823, -0.0121206, -2.26635e-05,
		     -0.0113449, -0.0122835, 0.999861, -0.002,
 		     0.0,        0.0,        0.0,      1.0]
      T_cn_cnm1:
        [0.9999473352, 0.007137018281, -0.007449203921, -0.1199447309,
 		    -0.007085288801, 0.9999494377, 0.007049787532, -0.001649370484,
  		  0.007499504247, -0.006995638145, 0.9999486994, -0.001381243423,
        0.0,         0.0,         0.0,             1.0]
      camera_model: pinhole
      distortion_coeffs: [0.593821, 0.199323, -0.604329, 0.270467]
      distortion_model: equidistant
      intrinsics: [245.988, 245.4165, 315.557, 203.7775]
      resolution: [640, 400]
      timeshift_cam_imu: 0.0
    T_imu_body:
      [1.0000, 0.0000, 0.0000, 0.0000,
      0.0000, 1.0000, 0.0000, 0.0000,
      0.0000, 0.0000, 1.0000, 0.0000,
      0.0000, 0.0000, 0.0000, 1.0000]
